invoices_app/views.py

```diff
@@ -540,31 +536,33 @@ def _send_email_when_change_invoice_status(request, invoice):
     activate(request.user.preferred_language)
 
 
-def _send_email_when_editing_invoice(instance, ap_user):
-    recipient_list = get_user_emails_by_tax_payer_id(instance.taxpayer.id)
-
-    users = User.objects.filter(email__in=recipient_list)
-    for user in users:
-        activate(user.preferred_language)
-
-        subject = str(EVENTBRITE_INVOICE_EDITED)
-        upper_text = str(INVOICE_EDIT_INVOICE_UPPER_TEXT.format(instance.invoice_number))
-        message = build_mail_html(
-            instance.taxpayer.business_name,
-            upper_text,
-            str(THANK_YOU),
-            DISCLAIMER,
-        )
-
-        _send_email(subject, message, [user.email])
-
-    activate(ap_user.preferred_language)
-
-
 def _send_email(
     subject,
     message,
     recipient_list
 ):
     if subject and message and recipient_list:
         send_email_notification.apply_async([subject, message, recipient_list])
+
+
+def get_upper_and_email_text(invoice: Invoice, request):
+    if invoice.status == '1':
+        return [INVOICE_STATUS_APPROVED_UPPER.format(invoice.invoice_number),
+                INVOICE_STATUS_APPROVED_EMAIL.format(invoice.invoice_number),
+                ]
+    elif invoice.status == '3':
```

Comment:
nit: if you are using return in each case of if. you could use if for the next cases.